### PR TITLE
chore(firebase-bigquery-export): added europe-central2 location to import script validation

### PR DESCRIPTION
**Pull request by dackers86**
PR to update the validation list for BigQuery `import` script.

- [x]  Added missing europe-central2
- [x] Re-ordered list alphabetically for easier future maintenance

fixes: https://github.com/firebase/extensions/issues/913

### CODE REVIEW

**Comment by dackers86** [comment]
### Testing

image

File: firestore-bigquery-export/scripts/import/src/index.ts [hunk 1]
```diff
@@ -69,32 +69,33 @@ const validateBatchSize = (value: string) => {
...
     "asia-northeast2",
     "asia-northeast3",
     "asia-southeast1",
-    "australia-southeast1",
     "asia-east1",
     "asia-northeast1",
-    "us",
+    "australia-southeast1",
     "eu",
+    "europe-west1",
```
Comment (russellwheatley) [inline firestore-bigquery-export/scripts/import/src/index.ts:82]: Nit, nit - Any chance you could also order numerically as well like in previous incarnation? i.e. "europe-west1", "europe-west2", etc.

**Review by russellwheatley — CHANGES_REQUESTED** [review]

File: firestore-bigquery-export/scripts/import/src/index.ts (hunk 1 above)
Comment (dackers86) [reply firestore-bigquery-export/scripts/import/src/index.ts:82]: This has been added as a separate update in https://github.com/firebase/extensions/pull/914/files

Comment (dackers86) [reply firestore-bigquery-export/scripts/import/src/index.ts]: @russellwheatley Thanks. Updated!

**Review by joehan — APPROVED** [review]

**Comment by alapeta** [comment]
Hello,

please add europe-central2 again as it disappeared after latest changes... :-(

**Pull request merged**